src/components/WebexInterstitialMeeting/WebexInterstitialMeeting.scss

```diff
@@ -14,6 +14,12 @@
     flex-direction: column;
   }
 
+  .interstitial-media-container {
+    display: flex;
+    position: relative;
```

Comment:
because this is now the reference for the banner, do we still need position: relative on `.media-container`?